### PR TITLE
Fixed Select README import example #267

### PR DESCRIPTION
**Pull request by filoscoder**
## Description

Fixed import example on readme file of Select component due to issue #265 

> Also, since I updated from 0.0.11 to 0.0.12 the import of "SwitchToggle" doesn't work anymore.
> image

I can't figure out why is happening this behavior please take a look at that issue!

## Related Issues

#265 

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Review by hyochan — APPROVED** [review]
🥇

**Comment by hyochan** [comment]
Everything looks good except the `CI`.

**Comment by filoscoder** [comment]
> Everything looks good except the `CI`.

oops! let me fix that

**Comment by hyochan** [comment]
> > Everything looks good except the `CI`.
> 
> oops! let me fix that

Update `jest snapshot` please. Still failing in `CI`.

File: main/SwitchToggle/index.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const styles = StyleSheet.create({
   },
 });
 
-function SwitchToggle(props: Props): React.ReactElement {
+const SwitchToggle: React.FC<Props> = (props): React.ReactElement => {
```
Comment (hyochan) [inline main/SwitchToggle/index.tsx:45]: `: React.ReactElement` is redundant in this case. That is covered in `React.FC`

**Pull request merged**